incydr/cli/cmds/cases.py

```diff
@@ -47,9 +71,9 @@
 @log_level_option
 @log_file_option
 @click.pass_context
-def cases(ctx, log_level, log_file):
+def cases(ctx: Context, log_level, log_file):
     """View and manage cases."""
-    init_client(ctx, log_level, log_file)
+    ...
```

Comment:
calling init_client at root command and here was causing options passed after `incydr` to get eaten. for example:

```
incydr --log-level debug cases list
```

would not keep log level at warn as the second init_client call was overriding it.

Reply:
Ahh, I only was calling that method within the subcommands because that would allow us to import `only` the subcommands if we ever wanted to, I think we maybe discussed doing it for the apps? 

If thats functionality we wanted to keep, maybe we could add a hidden option (like an `--init-client` flag) that would prompt the `init_client()` method